summary_workflows/quantification/quantification_dockers/q_consolidation/assessment_chart/assessment_chart.py

```diff
@@ -281,7 +327,7 @@ def print_chart(challenge_dir, summary_dir, challenge_type, classification_type)
         ax.get_yaxis().set_major_formatter(plt.FuncFormatter(lambda y, loc: "{:,}".format(int(y))))
 
     # set parameters for optimization
-    better = "top-right"
+    better = "top-left"
```

Comment:
Looks like we're still hard coding this variable instead of reading it from the aggregation object as @ninsch3000 suggested in the issue: https://github.com/iRNA-COSI/APAeval/issues/242
Is there a reason we're keeping it hardcoded here?

Reply:
good catch, forgot to add the changes here haha